src/libraries/System.Runtime/tests/System.Runtime.Extensions.Tests/System/Convert.ToHexString.cs

```diff
@@ -137,12 +189,33 @@ public static unsafe void ToHexString(byte[] input, string expected)
             Assert.Equal(expected, actual);
         }
 
+        [Theory]
+        [MemberData(nameof(ToHexStringTestData))]
+        public static unsafe void TryToHexString(byte[] input, string expected)
+        {
+            Span<char> output = stackalloc char[expected.Length];
```

Comment:
`stackalloc` with potentially unbounded size is problematic since it can lead to stackoverflow.

The lengths exercised by this test happen to be small enough. Regular `new char` would be more appropriate in this situation.